Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -80,7 +80,7 @@ class FontDescription {
     FontVariantNumericFraction variantNumericFraction() const { return static_cast<FontVariantNumericFraction>(m_variantNumericFraction); }
     FontVariantNumericOrdinal variantNumericOrdinal() const { return static_cast<FontVariantNumericOrdinal>(m_variantNumericOrdinal); }
     FontVariantNumericSlashedZero variantNumericSlashedZero() const { return static_cast<FontVariantNumericSlashedZero>(m_variantNumericSlashedZero); }
-    FontVariantAlternates variantAlternates() const { return m_variantAlternates; }
+    const FontVariantAlternates& variantAlternates() const { return m_variantAlternates; }
```

Comment:
Not a large type, not clear this is actually better. Passing small types by value is often faster than by reference. This looks like less than 2 words.

Reply:
This is a `Markable` that wraps a large object. I think this one does make sense to pass by reference.

Reply:
Agreed, I had looked correctly.